apps/server/src/shared/controller/transformer/index.ts

```diff
@@ -1,3 +1,4 @@
 export * from './string-to-boolean.transformer';
 export * from './decode-html-entities.transformer';
```

Comment:
Do we need decode-html-entities.transformer, or can it be removed?

Reply:
I think we still need the `@DecodeHtmlEntities` decorator in situations where we don't want to support rich text and want Vue to escape the output via `{{ }}`. Sometimes we have strings that were previously encoded by the legacy client. Encoding them again in Vue would not be a good thing.
https://github.com/hpi-schul-cloud/schulcloud-server/blob/2088f7a64d8d03ff9ad57ad315ea7801b3b81e57/apps/server/src/shared/controller/transformer/decode-html-entities.transformer.ts#L5